src/features/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: { value: '' },
+  reducers: {
+    checkStatus: (state, action) => ({
+      ...state,
+      value: action.payload,
+    }),
+  },
+});
+
+export const { checkStatus } = categoriesSlice.actions;
+
+export default categoriesSlice.reducer;
```

Comment:
- [ ] Also as mentioned above, kindly follow the duck pattern as it is part of project requirement